src/components/ciPipeline/SourceMaterials.tsx

```diff
@@ -132,23 +133,17 @@ export const SourceMaterials: React.FC<SourceMaterialsProps> = function (props)
                             </div>
                             <div className="mt-16 flex left">
                                 <div className="w-50 mr-8">
-                                    <label className="form__label mb-6 dc__required-field">
-                                        Source Type
-                                    </label>
+                                    <label className="form__label mb-6 dc__required-field">Source Type</label>
                                     <ReactSelect
                                         className="workflow-ci__source"
                                         placeholder="Source Type"
                                         isSearchable={false}
                                         menuPortalTarget={document.getElementById('visible-modal')}
-                                        options={props.ciPipelineSourceTypeOptions}
+                                        options={sourceTypeOptions}
                                         value={selectedMaterial}
                                         closeMenuOnSelect={true}
                                         onChange={(selected) => props?.selectSourceType(selected, mat.gitMaterialId)}
                                         isClearable={false}
-                                        isDisabled={
-                                            ciPipelineId !== '0' &&
-                                            (props.includeWebhookEvents ? !isProviderChanged : !!mat.id)
```

Comment:
Is removal of this expected?

Reply:
I need to make the source type editable ,so i need to remove the disabled attribute ,to make the source type editable